src/com/gmail/chernii/oleksii/ScheduledFutureExample.java

```diff
@@ -0,0 +1,43 @@
+package com.gmail.chernii.oleksii;
+
+import java.util.concurrent.*;
```

Comment:
Don't use a wild card in import. 
https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad